src/Server.php

```diff
@@ -10,117 +10,8 @@
 
 /**
  * Server Interface
+ * @deprecated Server is replaced by ServerInterface and will be removed in 3.0.
```

Comment:
As above, should be noted since when it's deprecated.

Reply:
Added since remark.